_pytest/logging.py

```diff
@@ -242,6 +242,8 @@ def __init__(self, config):
         The formatter can be safely shared across all handlers so
         create a single one for the entire test session here.
         """
+        self.log_level = get_actual_log_level(
+            config, 'log_level') or logging.WARNING
```

Comment:
We might consider to use `None` as default; then if `None`, we don't touch the [log level](https://github.com/segevfiner/pytest/blob/b9ddb253a55bc517f83a3e4a5bed5af70bb93ddf/_pytest/logging.py#L98) at all all.

Reply:
Note that if you pass `None` to `logger.setLevel` or `handler.setLevel` a TypeError is raised.

Reply:
Indeed, but actually I meant to check for `None` before calling `setLevel`.